src/modules/SignIn.js

```diff
@@ -0,0 +1,122 @@
+import { Formik } from "formik";
+import { useHistory } from "react-router-dom";
+import FormInput from '../FormInput';
+import FormButton from '../FormButton';
+
+
+function SignIn() {
```

Comment:
this and SignUp component have some validation logic and critical business logic as well
I think this is time to add tests for this done in `testing-library`. Please add tests for SignIn and SignUp components